patches/components-external_intents-android-BUILD.gn.patch

```diff
@@ -1,8 +1,16 @@
 diff --git a/components/external_intents/android/BUILD.gn b/components/external_intents/android/BUILD.gn
-index 530cfa2c449b94854ace00cc710818fb496d2a7e..4dab931ff4d955f8a04bfe7bdfce7cde9f37ebd8 100644
+index 530cfa2c449b94854ace00cc710818fb496d2a7e..6d06fdb8448349621a86d0c9ff5f0d88acbe0eac 100644
 --- a/components/external_intents/android/BUILD.gn
 +++ b/components/external_intents/android/BUILD.gn
-@@ -57,6 +57,7 @@ static_library("android") {
+@@ -29,6 +29,7 @@ android_library("java") {
+     "//ui/android:ui_java",
+     "//url:gurl_java",
+   ]
++  deps += brave_chrome_java_deps enable_bytecode_rewriter = true
```

Comment:
We don't need `deps` here, only `enable_bytecode_rewriter`. Also, I think we should find one place where we can patch it for all the components if it's possible.

Reply:
@samartnik  fixed at https://github.com/brave/brave-core/compare/fb01c0a448cb7a75dbb9729ec2c3b32d6c9f2c00..44239ddb3f5c8b79743cfa502297750261ae8f5b